src/paperless_mail/mail.py

```diff
@@ -190,6 +190,20 @@ def handle_mail_rule(self, M, rule):
         try:
             M.folder.set(rule.folder)
         except MailboxFolderSelectError:
+
+            self.log(
+                "error",
+                f"Unable to access folder {rule.folder}, attempting folder listing",
+            )
+            try:
+                for folder_info in M.folder.list():
+                    self.log("info", f"Located folder: {folder_info.name}")
+            except Exception:
+                self.log(
+                    "error",
+                    "Exception during folder listing, unable to provide list folders",
```

Comment:
```suggestion
            except Exception as e:
                self.log(
                    "error",
                    "Exception during folder listing, unable to provide list folders: " + str(e),
```
do we benefit from adding the exception text here?